### ISSUE TITLE
macOS: overly bright blue colours

### ISSUE THREAD

**Issue by AaronVanGeffen** [OpenRCT2/OpenRCT2#5894]
**OS:** macOS
**Version:** 0.1.0
**Commit/Build:** 1aed33192804541bd8e891595d64c7751d29b567

The background colours used by e.g. the settings screen appear a bit washed-out, as the screenshots below illustrate.

This occurs using the software renderer. Whether hardware backing is enabled or not appears to be unrelated.

**Screenshots:**
Title screen appears normal here:
![screen shot 2017-07-14 at 12 41 12](https://user-images.githubusercontent.com/604665/28209232-ee980784-6891-11e7-81e3-e3499ca7da51.png)
In scenario they appear washed out:
![screen shot 2017-07-14 at 12 41 21](https://user-images.githubusercontent.com/604665/28209233-ee9be3b8-6891-11e7-814a-3399ef8c4a03.png)

**Comment by IntelOrca** [comment]
Do you have any lightfx settings enabled?

**Comment by AaronVanGeffen** [comment]
Ah, I did! Disabling them resolves the issue, but only after I restarted the game. Thank you.

Related bug: the option for lightfx appears greyed-out when hardware backing isn't enabled, and isn't re-enabled when the drawing engine is changed. Only after re-opening the Settings window can the checkbox  finally be unticked.

**Comment by IntelOrca** [comment]
This is probably a duplicate of #5178.
